tests/conftest.py

```diff
@@ -106,7 +106,7 @@ def new_exception(msg, *_, **__):
         yield
 
 
-@pytest.fixture(autouse=True, scope="module")
+@pytest.fixture(autouse=True)
```

Comment:
Do we want to do this? How much runtime overhead for the tests is changing the scope back to the default (`function`) here?

Reply:
Never mind. This fixture is faster than a time traveling delorean.